.gitignore

```diff
@@ -84,3 +84,6 @@ Brewfile.lock.json
 
 # Ignore Test User Dashboard csv
 test_users.csv
+
+# Ignore Mac OS hidden file
+.DS_Store
```

Comment:
OS level files should be ignored in your global gitignore, not per repository. See the "Felt tip" here: https://stackoverflow.com/a/19299889/282720.

Reply:
thanks for the tip!